base/src/main/java/io/spine/base/Environment.java

```diff
@@ -78,19 +77,19 @@
  *
  *     static {
  *         Environment.instance()
- *                    .register(StagingEnvironmentType.type())
- *                    .register(LoadTestingType.type());
+ *                    .register(new StagingEnvironmentType())
```

Comment:
Let's skip `Type` here, just like we don't call it `TestsType` and `ProductionType`.